nemo/tron/checkpointing.py

```diff
@@ -25,7 +25,7 @@
 from logging import getLogger
 from pathlib import Path
 from time import time
-from typing import Optional
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
```

Comment:
Python now supports and defaults to `dict, list, tuple` as type hints. Can we use them?